third_party/move/move-vm/types/src/loaded_data/runtime_types.rs

```diff
@@ -1326,6 +1326,123 @@ impl TypeBuilder {
     }
 }
 
+#[derive(Default)]
+pub struct TypeParamMap<'a> {
```

Comment:
TBA: doc comments for struct. Would be great to also mention that it's for transaction argument construction.